<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpaceSwitcherConfiguration</class>
 <widget class="QWidget" name="SpaceSwitcherConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hl_selectedControl">
     <item>
      <widget class="QLabel" name="lbl_selectedControl">
       <property name="text">
        <string>Selected Control: None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_jsonContents">
     <item>
      <widget class="QLabel" name="lbl_jsonContents">
       <property name="text">
        <string>Control_Configuration attribute JSON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_generateDefaultJsonContents">
       <property name="text">
        <string>Generate Default JSON</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="te_jsonContents">
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="tabStopDistance">
      <double>40.000000000000000</double>
     </property>
     <property name="placeholderText">
      <string>JSON goes here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_set">
     <property name="text">
      <string>Set Space Configuration on Control</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_selectedObject">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Selected Object Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_getSelectedObjectName">
       <property name="toolTip">
        <string>Push this to set the below text box to the name of the selected object. You can copy paste that into the configutarion JSON above.</string>
       </property>
       <property name="text">
        <string>Get Selected Object Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="le_selectionName">
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="placeholderText">
      <string>Selected Object Name (Copy paste from here to the JSON)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
